hw3/03-form.js

```diff
@@ -1,10 +1,47 @@
-const http = require('http');
-const port = process.env.PORT || 5001;
+const http = require("http");
+const port = process.env.PORT || 5000;
+const staticServe = require("node-static");
```

Comment:
I didn't have node-static installed on the system so when I ran `node 03-form.js` I got a `MODULE_NOT_FOUND` error. You may want to consider adding a package.json to make it easier to get the modules working.

Once I got the module installed and set up, though, everything worked exactly as expected! Great work it looks fantastic!